src/tracy-modification.c

```diff
@@ -107,6 +107,8 @@ int tracy_inject_syscall_async(struct tracy_child *child, long syscall_number,
  */
 int tracy_inject_syscall_pre_start(struct tracy_child *child, long syscall_number,
         struct tracy_sc_args *a, tracy_hook_func callback) {
+    struct TRACY_REGS_NAME newargs;
```

Comment:
This fails on non-arm targets; because it's an unused variable when __arm__ is not defined. I'll try to think of something that doesn't require ifdef's all over the place in the main files. I can probably put it in the define below.